e2e/davinci-suites/src/basic.test.ts

```diff
@@ -85,3 +85,30 @@ test('ensure query params passed to start are sent off in authorize call', async
   const accessToken = await page.locator('#accessTokenValue').innerText();
   expect(accessToken).toBeTruthy();
 });
+
+test('Enter a bad username/password, then enter a good username/password', async ({ page }) => {
```

Comment:
I'm not seeing a submission of the "bad" credentials, so I'm a bit confused. What's the intent of this test?